openedx/features/student_certificates/views.py

```diff
@@ -156,3 +167,49 @@ def shared_student_achievements(request, certificate_uuid):
 
     response = render_to_response('shared_certificate.html', context)
     return response
+
+
+def download_certificate_pdf(request, certificate_uuid):
+    """
+    Convert user certificate image on S3 bucket to PDF file and download it for end user
+    :param request: HttpRequest obj
+    :param certificate_uuid: certificate unique id
+    :return: downloadable PDF file
+    """
+
+    url = get_certificate_image_url_by_uuid(certificate_uuid)
+
+    with requests.get(url, stream=True) as req:
+        if req.status_code != 200:
+            raise Exception("Unable to download certificate")
+
+        with TemporaryFile() as temp_file:
```

Comment:
please verify the path on which this file is saved temporarily, it should be` /tmp/`

Reply:
I have verified that using `NamedTemporaryFile`, also added param `dir='/tmp' `